parser/model/ddl.go

```diff
@@ -426,14 +426,18 @@ type JobMeta struct {
 
 // BackfillMeta is meta info of the backfill job.
 type BackfillMeta struct {
-	EndInclude bool   `json:"end_include"`
-	ErrMsg     string `json:"err_msg"`
+	PhysicalTableID int64  `json:"physical_table_id"`
+	IsUnique        bool   `json:"is_unique"`
```

Comment:
What is this field used for?

Reply:
There's a place in lighting for it, in the next PR: `bc.FinishImport(bfJob.EleID, bfJob.Meta.IsUnique, tbl)`